nesta_ds_utils/file_ops.py

```diff
@@ -1,53 +1,151 @@
-from io import BytesIO
-from typing import Union
+import io
+from typing import Union, List
 from pathlib import Path
 from xmlrpc.client import Boolean
 import zipfile
 import os
+import boto3
+from fnmatch import fnmatch
+import pandas as pd
 
 
 def _convert_str_to_pathlib_path(path: Union[Path, str]) -> Path:
```

Comment:
I know this is beyond the scope of this PR, but if this isn't used anywhere else, it could probably be rolled into `make_path_if_not_exist`

Reply:
Happy to address this in this PR.